lib/chef/provider/cron.rb

```diff
@@ -51,6 +67,9 @@ def load_current_resource
         @current_resource = Chef::Resource::Cron.new(@new_resource.name)
         @current_resource.user(@new_resource.user)
         @cron_exists = false
+        # Raise an execption if Chef can't read or update the crontab.
+        can_modify_cron?
```

Comment:
Ok, I'll have a look at this tomorrow,  I may ask for 'backup' if I don't get by myself how to switch between normal and why-run mode.

Still wondering if fixing the closed PR referenced in the issue would be better or not (overlooked the previous PR before submitting this one)